From: riku.voipio <riku.voipio@linaro.org>
Subject: Add gcc toolchain for arm64/linux
 This is needed for Chromium builds in debian/arm64 and matches 
 how arm, x86 and x86_64 also have both clang and gcc toolchains available.
Origin: upstream, https://chromium.googlesource.com/chromium/src.git/+/b9a43ab09791a1dcfd185141f71627c3b8d437db

Index: chromium-browser-54.0.2840.101/build/toolchain/linux/BUILD.gn
===================================================================
--- chromium-browser-54.0.2840.101.orig/build/toolchain/linux/BUILD.gn
+++ chromium-browser-54.0.2840.101/build/toolchain/linux/BUILD.gn
@@ -21,6 +21,24 @@ clang_toolchain("clang_arm64") {
   }
 }
 
+gcc_toolchain("arm64") {
+  toolprefix = "aarch64-linux-gnu-"
+
+  cc = "${toolprefix}gcc"
+  cxx = "${toolprefix}g++"
+
+  ar = "${toolprefix}ar"
+  ld = cxx
+  readelf = "${toolprefix}readelf"
+  nm = "${toolprefix}nm"
+
+  toolchain_args = {
+    current_cpu = "arm64"
+    current_os = "linux"
+    is_clang = false
+  }
+}
+
 gcc_toolchain("arm") {
   toolprefix = "arm-linux-gnueabihf-"
 
